conf/airframes/examples/bebop2_gvf.xml

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop2_gvf">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="bebop2">
+      <define name="STABILIZATION_INDI_G2_R" value="0.2784"/>
+    </target>
+
+    <target name="nps" board="pc">
+      <module name="fdm" type="jsbsim"/>
+      <module name="udp"/>
+      <define name="STABILIZATION_INDI_G2_R" value="0.0"/><!-- for jsbsim (rotor inertia is not modelled) -->
+      <!-- <define name="STABILIZATION_INDI_G2_R" value="0.20"/> --><!-- for gazebo -->
+    </target>
+
+	  <!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="bebop"/>
+    <module name="imu" type="bebop"/>
+    <module name="gps" type="ublox"/>
+    <module name="stabilization" type="indi_simple"/>
+    <module name="ins" type="ekf2"/>
+    <module name="gvf_module"/>
```

Comment:
Is this module the only difference with other a bebop 2 examples. If yes, the module can be loaded directly from the flight plan. It makes the FP more portable and avoid to add an extra airframe example that is very close to other ones.

Reply:
Yes, thanks for pointing it out, didn't know that possibility.